Mage.Sets/src/mage/sets/lorwyn/SylvanEchoes.java

```diff
@@ -0,0 +1,94 @@
+ *        conditions and the following disclaimer.
+ *
+ *     2. Redistributions in binary form must reproduce the above copyright notice, this list
+ *        of conditions and the following disclaimer in the documentation and/or other materials
+ *        provided with the distribution.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY BetaSteward_at_googlemail.com ``AS IS'' AND ANY EXPRESS OR IMPLIED
+ *  WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ *  FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL BetaSteward_at_googlemail.com OR
+ *  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ *  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ *  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ *  ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ *  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+ *  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *  The views and conclusions contained in the software and documentation are those of the
+ *  authors and should not be interpreted as representing official policies, either expressed
+ *  or implied, of BetaSteward_at_googlemail.com.
+ */
+package mage.sets.lorwyn;
+
+import java.util.UUID;
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.effects.common.DrawCardSourceControllerEffect;
+import mage.cards.CardImpl;
+import mage.constants.CardType;
+import mage.constants.Rarity;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+
+/**
+ *
+ * @author Styxo
+ */
+public class SylvanEchoes extends CardImpl {
+
+    public SylvanEchoes(UUID ownerId) {
+        super(ownerId, 237, "Sylvan Echoes", Rarity.UNCOMMON, new CardType[]{CardType.ENCHANTMENT}, "{G}");
+        this.expansionSetCode = "LRW";
+
+        // Whenever you clahs and you win, you may draw a card
+        this.addAbility(new SylvanEchoesTriggeredAbility());
+    }
+
+    public SylvanEchoes(final SylvanEchoes card) {
+        super(card);
+    }
+
+    @Override
+    public SylvanEchoes copy() {
+        return new SylvanEchoes(this);
+    }
+}
+
+class SylvanEchoesTriggeredAbility extends TriggeredAbilityImpl {
+
+    public SylvanEchoesTriggeredAbility() {
+        super(Zone.BATTLEFIELD, new DrawCardSourceControllerEffect(1), true);
+    }
+
+    public SylvanEchoesTriggeredAbility(final SylvanEchoesTriggeredAbility ability) {
+        super(ability);
+    }
+
+    @Override
+    public SylvanEchoesTriggeredAbility copy() {
+        return new SylvanEchoesTriggeredAbility(this);
+    }
+
+    @Override
+    public boolean checkEventType(GameEvent event, Game game) {
+        return event.getType() == GameEvent.EventType.CLASHED;
+    }
+
+    @Override
+    public boolean checkTrigger(GameEvent event, Game game) {
```

Comment:
Same comment as before but seeing the same code again I'm guessing you know more than I do. You're still not checking the clash was won but I suspect these event data values are doing that for you. If so then this is okay.